### DISCUSSION TITLE
what should be considered  when using llhls feature with cdn

### DISCUSSION THREAD

**Discussion by fastfading** [bluenviron/mediamtx#1858] [Questions]
looks like when the video size  became bigger,  but download time is still less than fragment duration.
the live stream won't be smoothy

I am not sure what is wrong. 
I am using the aws cdn .

**Comment by aler9** [comment]
Hello, in LL-HLS segments are splitted into parts. You can tune the duration of both:

```yml
hlsSegmentDuration: 1s
hlsPartDuration: 200ms
```

If the CDN or readers can't keep up with the video stream, you can either decrease the video bitrate or increase these two values, resulting in a higher latency.

[a comment by an automated account is not shown]
